apps/web/hooks/api/Profile/useProfileUpdate.ts

```diff
@@ -0,0 +1,26 @@
+import { apiHandler } from '@app/config';
+import { useMutation, useQueryClient } from 'react-query';
+
+interface IInfo {
+    slug: string;
+    uid: string;
+    name: string;
+}
+
+const updateProfile = async (info: IInfo) => {
+    return await apiHandler.patch('/user', {
+        displayName: info.name,
+        slug: info.slug,
+        uid: info.uid,
```

Comment:
passing uid in every request is very bad can we parse the uid from cookies like bearer token  ?

   ```
   token = jwt.sign(uid,"val")
   send this as a cookie and in backend dcrypt the cookie  etc etc
   
  ```

Reply:
Will remove the uid from Dto